dom.bs

```diff
@@ -6499,6 +6678,26 @@ null), and boolean <var>synchronousCustomElements</var> (default false):
  <li><p>Return <var>result</var>.
 </ol>
 
+<p>To <dfn>create an element internal</dfn> given a <a for=/>document</a> <var>document</var>, an
```

Comment:
I actually am not sure what effect the `noexport` attribute has, but it's used elsewhere in DOM for spec-internal stuff, so:

```suggestion
<p>To <dfn noexport>create an element internal</dfn> given a <a for=/>document</a> <var>document</var>, an
```

Also, I first thought this might be related to `ElementInternals`, so a name like "internal create element steps" or "inner create element steps" would help avoid that. Not a big deal though.

Reply:
I mostly stopped using that as it doesn't have an effect. Except maybe to emphasize but I don't think that's needed here.